### PR TITLE
[enhancement](stats) limit bq cap size for analyze task

### PR DESCRIPTION
**Pull request by Kikyou1997**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Kikyou1997** [comment]
run buildall

**Comment by Kikyou1997** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisTaskExecutor.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ public AnalysisTaskExecutor(int simultaneouslyRunningTaskNum) {
             executors = ThreadPoolManager.newDaemonThreadPool(
                     simultaneouslyRunningTaskNum,
                     simultaneouslyRunningTaskNum, 0,
-                    TimeUnit.DAYS, new LinkedBlockingQueue<>(),
+                    TimeUnit.DAYS, new LinkedBlockingQueue<>(StatisticConstants.TASK_QUEUE_CAP),
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/statistics/AnalysisTaskExecutor.java:50]: both auto analyze and manual analyze use `AnalysisTaskExecutor`. if u just change this capacity, what will happen when do manual analyze more than task queue cap?

**Comment by Kikyou1997** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by Jibing-Li — APPROVED** [review]

**Pull request merged**